.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "env": {
+    "test": {
+      "presets": [
+        "es2015",
+        "react"
+      ]
+    }
+  }
+}
```

Comment:
You have it already in webpack, it's probably not necessary to have additional .babelrc file. Webpack rule should be enough.